### PR TITLE
Enhance project README

### PR DESCRIPTION
**Pull request by dlsniper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 # honnef.co/go/tools
 
-`honnef.co/go/tools/...` is a collection of tools and libraries for
-working with Go code, including linters and static analysis.
+This is a collection of tools and libraries for working with Go code,
+including linters and static analysis.
+
+## Install instructions
+
+To install it or update it run:
```
Comment (shurcooL) [inline README.md]: What is being referred to as "it" here?

Comment (dlsniper) [inline README.md]: The library itself. Not sure how to make this better.

Comment (shurcooL) [inline README.md]: It it "a collection"? That seems reasonable.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
 # honnef.co/go/tools
 
-`honnef.co/go/tools/...` is a collection of tools and libraries for
-working with Go code, including linters and static analysis.
+This is a collection of tools and libraries for working with Go code,
+including linters and static analysis.
```
Comment (shurcooL) [inline README.md]: I personally liked the previous version of the paragraph, that mentions the import path pattern, better. Why do you think it's better to remove it?

Comment (dominikh) [inline README.md]: I'm with @shurcooL on this one.

File: README.md (hunk 1 above)
Comment (dominikh) [inline README.md]: `To install or update all of the tools, run`

**Comment by dominikh** [comment]
Closing this PR. Some of the changes aren't improvements, other changes are insignificant and will be made redundant by my work on staticcheck.io, and the `go get -u` change isn't worth waiting on this PR to be updated for.

**Pull request closed without merging**